addons/test_mail_full/static/tests/tours/message_actions_tour.js

```diff
@@ -7,8 +7,7 @@ registry.category("web_tour.tours").add("star_message_tour", {
             run: "hover && click #chatterRoot:shadow .o-mail-Message [title='Mark as Todo'] i.fa-star-o",
         },
         {
-            trigger: "#chatterRoot:shadow .o-mail-Message:contains(Test Message)",
-            run: "hover #chatterRoot:shadow .o-mail-Message [title='Mark as Todo'] i.fa-star.o-mail-Message-starred",
+            trigger: "#chatterRoot:shadow .o-mail-Message i.fa-star.o-mail-Message-starred",
```

Comment:
I don't know how much messages there are, but we lost the assertion on the correct message (the one containing (Test Message). Also, `i.fa-star` could be removed, `o-mail-Message-starred` is clear enough.

Reply:
There's only 1 message anyway, the `:contains()` is useless.
I kept mostly the same selector because it's out-of-scope of PR change, the shortening of selector is to not have eslint complaining with long line.